### DISCUSSION TITLE
Really Print the converted label on a Generic printer (not Zebra)

### DISCUSSION THREAD

**Discussion by rafamuz** [porrey/Virtual-ZPL-Printer#44] [Ideas]
I have a Label printer that does not recognizes ZPL/EPL language. So that it only prints RAW (Generic Text) instructions. When printing PDF, Word Documents, it prints directly normal like a Laser or Inkjet normal printer. So that I was trying to use this app to simulate a ZPL printer on my machine and then it sends it directly to my "generic chinese" printer, where it could finally print the label (converted from ZPL txt files). Is that currently possible?

**Comment by porrey** [comment]
The images are written to the image cache folder. You could pick them up from there and send them to a printer.

**Comment by rafamuz** [comment]
Yeah, but I need to automate it. Like. Monitoring this folder and always
that there is an image, it prints and then delete the image.

Atenciosamente,

Rafael Féria Müzel


Em sex., 5 de jan. de 2024 às 00:14, Daniel Porrey ***@***.***>
escreveu:

**Comment by porrey** [comment]
There is a Nuget package with the ID Labelary.Service that can be used within your application.

**Comment by porrey** [comment]
Another option is to look at an application to auto print:

https://www.technospot.net/blogs/how-to-continuously-auto-print-files-in-a-directory/#google_vignette

**Comment by rafamuz** [comment]
Yes it will work to me. But if I put the txt with ZPL commands in the
directory. My Chinese printer won't print it normally. Because I have a
printer that does not interpret neither ZPL nor epl commands. Only raw
impression. So that it must be converted to pdf or any other to be printed
directly. I don't know how I could do it...

Atenciosamente,

Rafael Féria Müzel

Em sáb, 6 de jan de 2024 12:11, Daniel Porrey ***@***.***>
escreveu:

**Comment by rafamuz** [comment]
Maybe you can program it to me. I'll pay. How much do you charge me for
doing this software?

Atenciosamente,

Rafael Féria Müzel

Em sáb, 6 de jan de 2024 12:11, Daniel Porrey ***@***.***>
escreveu:

**Reply by porrey** [reply]
It is possible. I would need to know more details, especially some details on the printer.

**Comment by rafamuz** [comment]
Yes. The printer only reads RAW. It's a generic one. I can send you the
driver it uses to print for you in order to check it. It's on my Dropbox

https://www.dropbox.com/scl/fo/zwyha0d8d5ciaydj5jf27/h?rlkey=ud0fsqoyftee20m1teh1puptl&dl=0


About brand is Knup, but it's a "generic" brand. It is used to be imported
from some Chinese bigger factory and they put their own brand.

What more info do you want me to send you? I can send you anything.

Atenciosamente,

Rafael Féria Müzel

Atenciosamente,

Rafael Féria Müzel

Em ter, 9 de jan de 2024 22:30, Daniel Porrey ***@***.***>
escreveu:

**Reply by porrey** [reply]
Is the printer on the network or USB connected?

**Comment by rafamuz** [comment]
USB

Atenciosamente,

Rafael Féria Müzel

Em dom, 14 de jan de 2024 14:15, Daniel Porrey ***@***.***>
escreveu:
